compiler/luci/pass/src/CircleOptimizer.cpp

```diff
@@ -301,6 +302,10 @@ void CircleOptimizer::optimize(loco::Graph *g) const
   {
     phase.emplace_back(std::make_unique<luci::FoldDequantizePass>());
   }
+  if (_options->query(Options::Algorithm::FoldFullyConnected))
+  {
+    phase.emplace_back(std::make_unique<luci::FoldFullyConnectedPass>());
+  }
```

Comment:
(optional) Just note for future. Options are getting bigger and code lines is also bigger accordingly. So, we may introduce a dedicated constant container of `Options::Algorithm` enum class.

It would be like this.
```cxx
std::unique_ptr<logo::Pass> make_pass(const Options::Algorithm &algo)
{
  switch(algo)
  {
    case Options::Algorithm::FoldFullyConnected:
      return std::make_unique<luci::FoldFullyConnectedPass>();
    // ..
  }
}

for(const auto &algo: ALGORITHM_BOOKS)
{
  if (_options->query(algo))
    phase.emplace_back(make_pass(algo));
}
```

Reply:
Good suggestion!